webapp/utils/constants.jsx

```diff
@@ -894,6 +894,9 @@ export const Constants = {
     MIN_HASHTAG_LINK_LENGTH: 3,
     CHANNEL_SCROLL_ADJUSTMENT: 100,
     EMOJI_PATH: '/static/emoji',
+    CHANGE_EVENT: 'changed',
+    RECENT_EMOJI_KEY: 'recentEmojis',
+    MAXIMUM_RECENT_EMOJI: 27,
```

Comment:
Ok, I'll move `CHANGE_EVENT` and `MAXIMUM_RECENT_EMOJI` back to `emoji_store.jsx`.